tests/test_PD015.py

```diff
@@ -0,0 +1,45 @@
+"""
+Test to check for `.merge()` method on pandas root object.
+
+Although the `.merge()` method is defined for both the pandas root object
+and the pandas DataFrame object, the preferred syntax is to use the DataFrame.
+
+The only difference in the methods is that for the pandas root method, an
+additional positional parameter is required to specify the dataframe to be
+merged.  The methods are differentiated in the AST by comparing the type
+of the base object.  A definitive distinction between these methods must
+check the type of the base object.
+
+    pd.merge(left, right, how='inner', on=None, ... )
+    df.merge(right, how='inner', on=None, ... )
+"""
+import ast
+
+from pandas_vet import VetPlugin
+from pandas_vet import PD015
+
+
+def test_PD015_pass_merge_on_dataframe():
+    """
+    Test that using .merge() on the dataframe does not result in an error.
+    """
+    statement = "df1.merge(df2)"
+    tree = ast.parse(statement)
+    expected = []
+
```

Comment:
While you are in the tests file, can you remove some of the empty lines (unless that is flake8 telling you to put them there)?

Reply:
Can do.  I was following the 'arrange-act-assert' testing pattern with a space between each of those phases.